pandas/tseries/frequencies.py

```diff
@@ -38,18 +38,20 @@ class FreqGroup(object):
     FR_NS = 12000
 
 
-US_RESO = 0
-MS_RESO = 1
-S_RESO = 2
-T_RESO = 3
-H_RESO = 4
-D_RESO = 5
+NS_RESO = 0
```

Comment:
They're used in some of the `@classmethod`s it turns out so need to stay for now